samples/bluetooth/central_iso/src/main.c

```diff
@@ -251,8 +251,10 @@ int main(void)
 	param.sca = BT_GAP_SCA_UNKNOWN;
 	param.packing = 0;
 	param.framing = 0;
-	param.latency = 10; /* ms */
-	param.interval = interval_us; /* us */
+	param.c_to_p_latency = 10; /* ms */
+	param.p_to_c_latency = 10; /* ms */
```

Comment:
Would probably make to have a value similar to `interval_us` we can use for both here, to avoid duplicate literal values